bool w_torque	# サーボのトルクON/OFF ON=1/OFF=0

bool b_left	# コントローラ十字キー 左
bool b_down	# コントローラ十字キー 下
bool b_right	# コントローラ十字キー 右
bool b_up	# コントローラ十字キー 上
bool b_start	# STARTボタン
bool b_R3	# R3 右スティック押し込み
bool b_L3	# L3 左スティック押し込み
bool b_select	# SELECTボタン
bool b_square	# □ボタン
bool b_cross	# ×ボタン
bool b_circle	# ○ボタン
bool b_triangle	# △ボタン
bool b_R1	# R1ボタン
bool b_L1	# L1ボタン
bool b_R2	# R2ボタン
bool b_L2	# L2ボタン
int32 s_lx	# 左スティック左右方向 (-128~127)
int32 s_ly	# 左スティック前後方向 (-128~127)
int32 s_rx	# 右スティック左右方向 (-128~127)
int32 s_ry	# 右スティック前後方向 (-128~127)